src/main/resources/templates/profile.html

```diff
@@ -0,0 +1,129 @@
+                alt="sign up">
+          </a>
+        </div>
+      </div>
+
+      <!-- header search -->
+
+      <div class="header__search">
+
+      </div>
+
+      <!-- #### -->
+
+    </div>
+  </header>
+
+  <main class="main">
+
+    <!-- content of main -->
+
+    <div class="container">
+      <div class="content-profile content">
+
+        <div class="content-profile__column content-column">
+
+          <div class="content-registration__title">
+            <h2>Profile</h2>
+          </div>
+
+          <form action="#" method="POST" class="registration profile-registration">
+            <div class="registration__fields-wrapper">
+              <div class="registration__fields profile-registration__fields">
+                <label class="registration__label">
+                  <input class="registration__input" type="email" name="email" placeholder="E-mail" required>
+                </label>
+                <label class="registration__label">
+                  <input class="registration__input" type="text" name="discogsUserName" placeholder="Discogs User Name">
+                </label>
+              </div>
+            </div>
+            <label class="registration__submit-label">
+              <input class="registration__submit-input" type="submit" value="EDIT">
+            </label>
+          </form>
+
```

Comment:
This is profile page. Could you rename common classes from registration* to, for example, user*?